scripts/travis-install

```diff
@@ -15,10 +15,20 @@ then
     sudo apt-get autoremove
     sudo apt-get autoclean
     sudo rm -rf /var/lib/mysql
-    sudo truncate -s 0 /var/log/mysql/error.log
 
+    # Install Elasticsearch
+    wget https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.7.2.deb
+    sudo dpkg -i elasticsearch-1.7.2.deb
+    
```

Comment:
As I mentioned earlier with the first comment, according to the [doc](https://docs.travis-ci.com/user/trusty-ci-environment/#Data-Stores), elasticsearch is not available in ubuntu 14.04 trusty builds. so need to install the elasticsearch.